src/main/java/net/neoforged/neoforge/attachment/DataAttachmentOps.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) NeoForged and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.neoforged.neoforge.attachment;
+
+import com.mojang.serialization.DataResult;
+import com.mojang.serialization.DynamicOps;
+import com.mojang.serialization.codecs.RecordCodecBuilder;
+import net.minecraft.core.HolderLookup;
+import net.minecraft.resources.DelegatingOps;
+import net.minecraft.resources.RegistryOps;
+import net.minecraft.util.ExtraCodecs;
+
+public class DataAttachmentOps<T, TOwner> extends DelegatingOps<T> {
```

Comment:
This doens't extend `RegistryOps`, so codecs that do an `instanceof` check to see if they're using registry ops will fail when using this codec.